plugins/modules/monitoring/icinga2_host.py

```diff
@@ -282,7 +282,7 @@ def main():
                     if ret['code'] == 200:
                         changed = True
                     else:
-                        module.fail_json(msg="bad return code deleting host: %s" % (ret['data']))
+                        module.fail_json(msg="bad return code deleting host: %s" % (ret['code']))
```

Comment:
```suggestion
                        module.fail_json(msg="bad return code deleting host: %s %s" % (ret['code'], ret['data']))
```

Reply:
@felixfontein how bout something like this to maybe clarify which code is which if you only get one?

```suggestion
                        module.fail_json(msg="bad return code (%s) deleting host: '%s'" % (ret['code'], ret['data']))
```

Reply:
@dericcrago that would be even better!

Reply:
I adjusted the other two suggestions approriately, feel free to commit yours and them if you think it's good.